core/src/test/scala/org/apache/spark/util/MutableURLClassLoaderSuite.scala

```diff
@@ -19,9 +19,15 @@ package org.apache.spark.util
 
 import java.net.URLClassLoader
 
+import scala.collection.JavaConverters._
+
 import org.apache.spark.{SparkContext, SparkException, SparkFunSuite, TestUtils}
 
-class MutableURLClassLoaderSuite extends SparkFunSuite {
+import org.scalatest.Matchers
```

Comment:
nit: these imports should come before `org.apache.spark`. Also, nuke one of the empty lines before the class declaration.